### ISSUE TITLE
show status of meter (green, red, grey) also in group list and meter list

### ISSUE THREAD

**Issue by evaklopp** [buzzn/scrum#101]
**User story**

As a Stefan
I want to have an quick overview of all the meters
so that I know if my system is running well.

---

**Definition of done**

- [ ] ...

---

**Technical details**

_Frontend_

- [ ] ...

_Backend_

- [ ] ...

**Comment by mkristian** [comment]
@evaklopp @dottorer FYI: the red dot means that the connected meter never send any data to discovergy and stays red until the first reading gets send to discovergy. green means that there readings from discovergy. maybe we need to refine this green to have recent readings from discovergy and not that the last reading was 2 month ago ?

**Comment by evaklopp** [comment]
@mkristian yes, good idea. The green should say that everything is currently running well. So wee need a timespan for this? How about the same 15 secs we use for the bubbles? or is that too much?

**Comment by evaklopp** [comment]
from a design point of view connected to #99

**Comment by eharit** [comment]
@evaklopp @dottorer The 'meter list' in the title refers to the 'register list'?

**Comment by dottorer** [comment]
@eharit Yep, register list that is.

**Comment by dottorer** [comment]
I am unsure what the 'Meter Status' field on the group list would be based on. For a whole group, we cannot really derive a 'meter status'. We might need to clarify first what status is important to us here. I thought it would show the red triangle if we are missing data to operate this group correctly.

Let's check this with @evaklopp what she had in mind.

**Comment by evaklopp** [comment]
@eharit  @dottorer  

**in the case of the group list:** 
**green:** all meters are connected correctly, even if a group has some "grey" non smart meters. All good=green. 

**grey:** a group has NO smart meters at all. Do we have this case? 

**red:** if only one smart meter of a group is in trouble, the status in the group list turns to red. 
@mkristian let´s define the exact conditions for a "smart meter/register in trouble", I´m not clear about that at the moment. 

no status: could be just a "-" for no installed meters/registers in case of a new group.

Xperience: we could use the warning-triangle as well, but from a user point of view we should use only one warning-sign: either the red dot or the triangle. 
The colored dots fit to a visual status which is limited to defined cases like we have with the meters. 
The triangel could be more than that: missing data, error, bug, everything what is wrong. So I would go for the dots in this case. We must be careful with the "status of things" in our system, we have a lot.

**Comment by dottorer** [comment]
@evaklopp I am not sure this feature is making sens on the group level, at least at this point. Pho does not have this ability today and I am not sure it is his highest value feature (pls check with him).

We can have a mix of meter types (smart/connected, smart/unconnected, non-smart) which in the case of smart/connected can have a success/failure state. Until we understand the landscape out there better and also know what Pho would do in terms of action, I would not aggregate this on the group. WE already have the triangle for data consistency/completeness on the group, which makes more sense to me.

However, this leaves us with the need for a clean status on the individual register level. This is something we can discuss 1-1 in the next days.
